<launch>
  <param name="omni1_description" command="$(find xacro)/xacro.py '$(find raw_omni_description)/robots/omni1.urdf.xacro'" />
  <group ns="omni1">
    <node name="omni1" pkg="raw_omni" type="raw_omni" output="screen">
      <param name="omni_name" type="string" value="omni1" />
      <param name="omni_serial" type="string" value="50823300018" />
    </node>

    <node name="omni1_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
      <param name="publish_frequency" type="double" value="400.0" />
      <remap from="joint_states" to="omni1_joint_states" />
      <remap from="robot_description" to="omni1_description" />
    </node>

    <node pkg="tf" type="static_transform_publisher" name="omni1_sensable_publisher" args="-0.17 0 0.1 -1.57 0 1.57 omni1_base omni1_sensable 100" />
    <node pkg="tf" type="static_transform_publisher" name="omni1_base_publisher" args="0 0 0 0 0 0 map omni1_base 100" />
  </group>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find raw_omni)/cfg/single_omni.rviz" required="true" />
</launch>
